index.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Youtube Hompage</title>
+    <link rel="stylesheet" href="./css/style.css">
+    <link rel="stylesheet" href="./css/all.css">
+</head>
+<body>
+    <div class="container">
+        
+        <header class="header">
+            <div class="left-nav">
+                <i id="menu" class="fas fa-bars"></i>
+                <a href="/"><img src="img/youtube_logo.png" alt="Youtube Home" title="Youtube Home"></a>
+            </div>
+            <div class="center-nav">
+                <form action="#">
+                    <input type="text" placeholder="Search" />
+                    <button><i class="fas fa-search"></i></button>
+                </form>
+                <i id="mic" class="fas fa-microphone"></i>
+            </div>
+            <div class="right-nav">
+                <i class="fa fa-video-camera" aria-hidden="true"></i>
+                <i class="fa fa-th" aria-hidden="true"></i>
+                <i class="fa fa-bell" aria-hidden="true"></i>
+                <i class="fas fa-user-circle"></i>
+            </div>
```

Comment:
Please it would be better to use the semantic `nav` tag instead of a `div` for all the `nav` items and also use `ul` tags inside every `nav` since they contain lists. Thank you